src/main/java/com/hotels/hcommon/hive/metastore/client/tunnelling/MetastoreTunnel.java

```diff
@@ -96,4 +96,8 @@ public String getStrictHostKeyChecking() {
   public void setStrictHostKeyChecking(String strictHostKeyChecking) {
     this.strictHostKeyChecking = strictHostKeyChecking;
   }
+
+  public boolean isStrictHostKeyChecking() {
+    return strictHostKeyChecking.toLowerCase().equals("yes");
```

Comment:
Maybe: `"yes".equalsIgnoreCase(strictHostKeyChecking)` to be completely null safe?